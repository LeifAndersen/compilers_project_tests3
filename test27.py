a = 3
def b():
 a = 4
 def c():
  return a
 c()

