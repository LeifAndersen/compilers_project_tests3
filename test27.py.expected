(program
 (define b (void))
 (define a (void))
 (set-global! a 3)
 (set-global!
  b
  (lambda ()
    (call/ec
     (lambda (return)
       (let ((a (void)) (c (void)))
         (set! a 4)
         (set! c (lambda () (call/ec (lambda (return) (let () (return a))))))
         (c)))))))

